### DISCUSSION TITLE
Text edit support

### DISCUSSION THREAD

**Discussion by niksite** [chrislemke/ChatFred#86] [Ideas]
Recently, OpenAI introduced a new API endpoint for text processing: https://platform.openai.com/docs/api-reference/edits/create

Could you please implement support for it in ChatFred?

It would be so great to be able to select any text, press a hot key, type (or select from a list?) something along the lines of "rewrite in bussiness english" or "translate to french", hit enter and get the result?

**Comment by chrislemke** [comment]
Hey @niksite! 
That actually sounds pretty interesting. At the moment I am still working on some other features and bug fixes. Maybe you want to create a pull request?

**Comment by skynet01** [comment]
Any updates on this? :)

**Comment by chrislemke** [comment]
Hey, @skynet01!
Not really. Currently, I am working on something different. Maybe someone wants to contribute?
